tests/pass-dep/libc/libc-env.rs

```diff
@@ -0,0 +1,98 @@
+//@ignore-target: windows
+//@compile-flags: -Zmiri-disable-isolation
+
+use std::ffi::CStr;
+
+#[path = "../../utils/mod.rs"]
+mod utils;
+
+fn test_getenv() {
+    // PATH should exist and not be null
+    let s = unsafe { libc::getenv(b"PATH\0".as_ptr().cast()) };
+    assert!(!s.is_null());
```

Comment:
Please check `MIRI_ENV_VAR_TEST` instead, which is set by the test harness so it should always have the value "0".